### PR TITLE
CDA: route para cda sisa

### PR DESCRIPTION
**Pull request by plammel**
### Requerimiento

https://proyectos.andes.gob.ar/browse/EP-195
### Funcionalidad desarrollada 

1.  Route para post de cda de sisa
2. 
3. 

### UserStories llegó a completarse

- [ ] Si
- [ ] No

### Requiere actualizaciones en la base de datos

- [ ] Si
- [ ] No

### CODE REVIEW

File: modules/cda/routes/cda.ts [hunk 1]
```diff
@@ -23,103 +24,33 @@ router.post('/create', cdaCtr.validateMiddleware, async (req: any, res, next) =>
...
+router.post('/create/sisa-covid', cdaCtr.validateMiddleware, async (req, res, next) => {
+    if (!Auth.check(req, 'cda:post')) {
+        return next(403);
+    }
 
-        const dataPaciente = req.body.paciente;
-        const dataProfesional = req.body.profesional;
+    const { documento, sexo } = req.body.paciente;
+    const cda = req.body.cda;
+    const pacientes = await PacienteCtr.search({ documento, sexo }); // Identificamos el registro de paciente
```
Comment (negro89) [inline modules/cda/routes/cda.ts:40]: Para una busqueda por parametros agregaría { activo: true }, para evitar traer pacientes inactivos que puedan ser encontrados primero.

Comment (negro89) [inline modules/cda/routes/cda.ts:40]: Para una busqueda por parametros agregaría { activo: true }, para evitar traer pacientes inactivos que puedan ser encontrados primero.
Se admiten pacientes no validados?

**Review by negro89 — APPROVED** [review]

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6280](http://condorpiedra.andes.gob.ar:85/6280/mochawesome.html)
CYPRESS RUN: [5173](https://dashboard.cypress.io/projects/xr7gft/runs/5173/specs)
TEST START: 2022-04-26T11:27:34.521Z
TOTAL: 406
SUCCESS: 395
FAIL: 10
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6325](http://condorpiedra.andes.gob.ar:85/6325/mochawesome.html)
CYPRESS RUN: [5216](https://dashboard.cypress.io/projects/xr7gft/runs/5216/specs)
TEST START: 2022-05-03T03:31:55.467Z
TOTAL: 406
SUCCESS: 404
FAIL: 1
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6344](http://condorpiedra.andes.gob.ar:85/6344/mochawesome.html)
CYPRESS RUN: [5229](https://dashboard.cypress.io/projects/xr7gft/runs/5229/specs)
TEST START: 2022-05-03T19:00:50.447Z
TOTAL: 353
SUCCESS: 341
FAIL: 11
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6365](http://condorpiedra.andes.gob.ar:85/6365/mochawesome.html)
CYPRESS RUN: [5250](https://dashboard.cypress.io/projects/xr7gft/runs/5250/specs)
TEST START: 2022-05-05T10:52:24.872Z
TOTAL: 406
SUCCESS: 391
FAIL: 14
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6368](http://condorpiedra.andes.gob.ar:85/6368/mochawesome.html)
CYPRESS RUN: [5253](https://dashboard.cypress.io/projects/xr7gft/runs/5253/specs)
TEST START: 2022-05-05T11:35:10.282Z
TOTAL: 406
SUCCESS: 405
FAIL: 0
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6456](http://condorpiedra.andes.gob.ar:85/6456/mochawesome.html)
CYPRESS RUN: [5338](https://dashboard.cypress.io/projects/xr7gft/runs/5338/specs)
TEST START: 2022-05-17T14:32:36.553Z
TOTAL: 406
SUCCESS: 402
FAIL: 3
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6457](http://condorpiedra.andes.gob.ar:85/6457/mochawesome.html)
CYPRESS RUN: [5339](https://dashboard.cypress.io/projects/xr7gft/runs/5339/specs)
TEST START: 2022-05-17T14:54:57.831Z
TOTAL: 381
SUCCESS: 372
FAIL: 8
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6519](http://condorpiedra.andes.gob.ar:85/6519/mochawesome.html)
CYPRESS RUN: [5400](https://dashboard.cypress.io/projects/xr7gft/runs/5400/specs)
TEST START: 2022-05-30T15:08:46.769Z
TOTAL: 402
SUCCESS: 398
FAIL: 3
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6520](http://condorpiedra.andes.gob.ar:85/6520/mochawesome.html)
CYPRESS RUN: [5401](https://dashboard.cypress.io/projects/xr7gft/runs/5401/specs)
TEST START: 2022-05-30T15:50:28.068Z
TOTAL: 329
SUCCESS: 310
FAIL: 18
SKIPPED: 1
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6521](http://condorpiedra.andes.gob.ar:85/6521/mochawesome.html)
CYPRESS RUN: [5402](https://dashboard.cypress.io/projects/xr7gft/runs/5402/specs)
TEST START: 2022-05-30T16:15:26.010Z
TOTAL: 406
SUCCESS: 405
FAIL: 0
SKIPPED: 1

**Pull request merged**